projects/opencv/build.sh

```diff
@@ -26,6 +26,12 @@ popd
 
 for fuzzer in imdecode_fuzzer imread_fuzzer; do
 $CXX $CXXFLAGS -lFuzzingEngine $fuzzer.cc -std=c++11 \
-  $(pkg-config --static --libs --cflags $WORK/lib/pkgconfig/opencv4.pc) \
-  -o $OUT/$fuzzer
+-I/work/include/opencv4/opencv -I/work/include/opencv4 -L/work/lib \
```

Comment:
You should replace all `/work` paths with `$WORK`.

Reply:
Good catch, done.